posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -252,6 +277,9 @@ def group_fingerprints(self, groups):
                 exprs.append(ast.Constant(value=fp))
         return ast.Array(exprs=exprs)
 
+    def extracted_exception_list_property(self, property):
+        return parse_expr(f"JSON_VALUE(any(properties.$exception_list), '$[0].{property}')")
```

Comment:
I tried writing this using the ast but the parser wasn't happy. Anyone have thoughts as to why this wasn't working... nvm I tried running what I was about to type here and it worked 🤷